src/inventory/disks.go

```diff
@@ -55,20 +71,21 @@ func (d *disks) getDisks() []*models.Disk {
 		return ret
 	}
 	for _, disk := range blockInfo.Disks {
-		if disk.IsRemovable || disk.BusPath == ghw.UNKNOWN {
+		if disk.IsRemovable || disk.SizeBytes == 0 ||
```

Comment:
Why both BusType and StorageController?  Are there cases we saw it is needed? Do we want to list disks if only one of them exist?

Reply:
In the case of a dm device they are both unknown, so it's safer to filter on less conditions (I'd rather see extra disks than fewer).  As we learn more we'll be able to fine tune more.